client-lib/src/main/java/com/ibm/ws/repository/resources/internal/RepositoryResourceImpl.java

```diff
@@ -161,10 +162,16 @@ private RepositoryWriteableClient getWritableClient() throws RepositoryOperation
         if (_client instanceof RepositoryWriteableClient) {
             return (RepositoryWriteableClient) _client;
         } else {
-            throw new RepositoryOperationNotSupportedException("The backend does not support write operations", _repoConnection);
+            throw new RepositoryOperationNotSupportedException("The backend does not support write operations or no connection has been specified", getRepositoryConnection());
```

Comment:
If a connection has not been specified a _client won't have been created, so _client will be null and therefore not an instance of RepositoryWritabelClent